pkg/roachprod/roachprod.go

```diff
@@ -2376,3 +2376,48 @@ func createAttachMountVolumes(
 	}
 	return nil
 }
+
+// CreateLoadBalancer creates a load balancer for the SQL service on the given
+// cluster. Currently only supports GCE.
+func CreateLoadBalancer(
+	ctx context.Context,
+	l *logger.Logger,
+	clusterName string,
+	secure bool,
+	virtualClusterName string,
+	sqlInstance int,
```

Comment:
`sql instance` is the offset on a VM itself, so if we ever have a deployment of two tenant (of the same tenant) instances per node, you'll want to specify the sql instance offset.

The `cli` should be passing this though. Thanks for catching, I'll update that.

Reply:
Oh, I see what you mean. So, effectively, only one of the instances in the node can be part of the load balancing. 

(Related aside: I wonder if there's ever a use-case to start multiple instances of the same tenant on the same node, especially since we don't provide any kind of resource isolation. We might have been able to do away with "sql instances" altogether. But that's a different conversation.)

Reply:
@renatolabs I'm all for removing this. Add unnecessary complexity, and not sure if we gain any coverage from testing it. I remember the other option was to hide it in a struct, but that only slightly reduces its footprint.